src/gui/keymap.cpp

```diff
@@ -921,6 +921,31 @@ static QHash<QString, Key> getMap(KeyMap::Model model, KeyMap::Layout layout){
         map["lock"].height = 8;
         break;
     }
+    case KeyMap::K70_CORE_RGB:{
+            map = getMap(KeyMap::K70, layout);
+            map.remove("mute");
+            map["lock"] = {nullptr, "Lock", "lock", 20, 2, NS, false, true}; // F1 alternate
+            map["profswitch"] = {nullptr, "Profile Switch", "profswitch", 32, 2, 11, 11, false, true}; // F2 alternate
+            map["light"] = {nullptr, "Light", "light", 50, 2, NS, false, true}; // F3 and F4 alternate
+            map["stop"] = {nullptr, "Stop", "stop", 76, 2, 11, 11, false, true}; // F5 alternate
+            map["prev"] = { nullptr,  "Previous", "prev", 88, 2, 11, 11, false, true}; // F6 alternate
+            map["play"] = { nullptr,  "Play", "play", 100, 2, 11, 11, false, true}; // F7 alternate
+            map["next"] = { nullptr,  "Next", "next", 112, 2, 11, 11, false, true}; // F8 alternate
+            map["ctrlwheelb"] = {nullptr, "Wheel", "ctrlwheelb", 168, 2, NS, false, true}; // F12 alternate
+
+            // This is the media button that defaults to play
+            // We already have a play Fn button so we want to map it to something else so it can be programmed
+            map["g1"] = { nullptr,  "G1", "g1", 227, 14, 16, 11, false, true};
```

Comment:
I don't understand this.

Reply:
![image](https://github.com/user-attachments/assets/36761382-2197-48bf-9146-b4f0a0005443)

It's the key I've marked with a red box. It sends key code 124 or play/pause by default. We already have a play/pause button on this keyboard though (Fn + F7) and the Corsair software itself on Mac/Windows doesn't handle this button as a play/pause button but instead as a macro to store lighting profiles in the keyboard's memory. I just set it to something so that it can be remapped in to a useful key since it's intended function won't work. If there is a better something, tell me and I'll set it to that.